src/main/resources/reference.conf

```diff
@@ -1665,6 +1665,37 @@ resourceTypes = {
     allowLeaving = false
     reuseIds = true
   }
+
+  azure_private_endpoint_access = {
```

Comment:
`azure_managed_identity`?

Reply:
Are we representing an identity, or an endpoint, or the thing on the other side of the endpoint?